release-notes/README.md

```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## November 5, 2021
+
+ ### iOS AEPMobileServices 3.0.3
+
+ * Fixes crash using an immutable dictionary when handling attribution data.
```

Comment:
Rewording suggestion: Fixes crash when using an immutable dictionary for handling attribution data.